### DISCUSSION TITLE
Core Web vitals monitoring

### DISCUSSION THREAD

**Discussion by adiig7** [nuxt-community/sentry-module#707] [General]
I am trying to look for core web vitals in the documentation but can't find anything for that. I have integrated performance monitoring but have nothing for core web vitals. How do we do that for Nuxt2? Or `tracing=true` would be more than enough to get it working? @rchl

**Comment by rchl** [comment]
I believe `tracing: true` should be enough.

Alternatively could try
```ts
        tracing: {
            browserTracing: {
                enableInp: true,
            },
        },
```

because `enableInp` is not by default enable in SDK v7 but I think it's just small part of the functionality.

Also note this from the documenation:

**This feature is available only if your organization is on a [Business or Enterprise plan](https://sentry.io/pricing/).**
